settings.gradle

```diff
@@ -4,9 +4,15 @@ include 'tracing'
 include 'tracing-api'
 include 'tracing-benchmarks'
 include 'tracing-demos'
-include 'tracing-jaxrs'
-include 'tracing-jersey'
 include 'tracing-okhttp3'
-include 'tracing-servlet'
 include 'tracing-test-utils'
-include 'tracing-undertow'
+
+withJakarta('tracing-servlet')
+withJakarta('tracing-undertow')
```

Comment:
Is there any jakarta involved with tracing-undertow?

Reply:
Not anymore.